orchestrator/core/seeds/seed_auto_agent.py

```
@@ -87,64 +87,89 @@ def _upsert_platform_management_skill(db: Session) -> Skill | None:
+    # on hot paths across workers). The xact-scoped advisory lock releases with
+    # the surrounding transaction; the IntegrityError fallback covers the race
+    # against the live UNIQUE(name) WHERE workspace_id IS NULL index.
     try:
-        skill = db.query(Skill).filter(
-            Skill.name == "platform-management",
-            Skill.skill_source == "builtin-core",
-        ).first()
+        db.execute(_sql_text(
+            "SELECT pg_advisory_xact_lock(hashtext('seed:platform-management'))"
+        ))
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)